### PR TITLE
Fix for the forum input not being on the page

### PR DESCRIPTION
**Pull request by atainton**
If the input for the forum isnt on the page i have added a method to get the id or d from the params

### CODE REVIEW

File: amd/src/mention_users.js [hunk 1]
```diff
@@ -32,7 +32,11 @@ define(['jquery', 'local_mention_users/tribute'], function($) {
   module.init = function() {
 
     var reply_id = $('input[name=reply]').val();
-    var forum_id = $('input[name=forum]').val();
+    if ($('input[name=forum]').length > 0) {
+      var forum_id = $('input[name=forum]').val();
+    }else{
+      var forum_id = getParameterByName('d');
```
Comment (davidjnorth) [inline amd/src/mention_users.js]: Are you sure the d is the forum id? I would guess it would be the discussion id but I could easily be wrong.

**Comment by RitterNorbert** [comment]
👍

File: amd/src/mention_users.js (hunk 1 above)
Comment (atainton) [inline amd/src/mention_users.js]: http://staging.onlinecampus.getsmarter.ac/mod/hsuforum/discuss.php?d=68

Comment (atainton) [inline amd/src/mention_users.js]: this is the page that doesnt have the input and the param is d

Comment (davidjnorth) [inline amd/src/mention_users.js]: 👍

**Pull request merged**